features/step_definitions/plans/application_plans_steps.rb

```diff
@@ -139,7 +139,7 @@
 
 Given "the application plan is published" do
   @plan.publish!
-  assert @plan.reload.published?
+  assert @plan.published?
```

Comment:
why remove `reload`? Seems like a way to make sure it is checking actual database state, not just local object state.